docs-src/go/how-to-use-worker-versioning-in-go.md

```diff
@@ -115,3 +115,16 @@ var yourActivityResult YourActivityResultType
 err := workflow.ExecuteActivity(ctx, YourActivityDefinition, yourActivityParam).Get(ctx, &yourActivityResult)
 // ...
 ```
+
+#### Specifying versions for Continue-As-New
+
+When using the Continue-As-New, use the `WithWorkflowVersioningIntent` context modifier.
+
+The function `WithWorkflowVersioningIntent` sets the `VersioningIntentDefault` before constructing a `ContinueAsNewError` with `NewContinueAsNewError`:
```

Comment:
```suggestion
The function `WithWorkflowVersioningIntent` sets `VersioningIntentDefault` before constructing a `ContinueAsNewError` object with `NewContinueAsNewError`:
```
Or similar, per https://developers.google.com/style/code-in-text#grammatical-treatment-of-code-elements.